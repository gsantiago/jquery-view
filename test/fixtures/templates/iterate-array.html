<ul>
  <li :repeat="item in items">
    {{$index}}: {{item}}
  </li>
</ul>