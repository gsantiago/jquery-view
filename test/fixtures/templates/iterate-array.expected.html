<ul>
  <li>
    0: Banana
  </li><li>
    1: Orange
  </li><li>
    2: Apple
  </li><li>
    3: Pear
  </li>
</ul>